src/vl-pager.js

```diff
@@ -229,6 +229,10 @@ export class VlPager extends VlElement(HTMLElement) {
 
   _current_pageChangedCallback(oldValue, newValue) {
     this._update();
+    if (oldValue && newValue != oldValue) {
```

Comment:
Is die check op oldValue om te voorkomen dat er bij de allereerste keer dat hij in deze callback komt (bij het initialiseren van het element op de pagina), een event wordt uitgestuurd?

Reply:
yes

Reply:
Lijkt me inderdaad overbodig?

Reply:
dan gaat de afnemer toch zelf state moeten bij houden? want dan weet die niet of die 1e change al echt nen change is of niet? dus ik zou dit laten zitten, ik vind dat niet een "change"

Reply:
Inderdaad, goed argument. Hij kan op het `click` event luisteren dan. Dus resolve.